### DISCUSSION TITLE
Property Names Don't Render

### DISCUSSION THREAD

**Discussion by jcoughlin11** [thegeeklab/hugo-geekdoc#544] [Q&A]
I'm using the [properties shortcode](https://geekdocs.de/shortcodes/propertylist/) in one of my markdown files, but the names of the properties are not being displayed, only a number (0, 1, 2, ...).

I copy and pasted the `data/properties/demo.yaml` file into my site thinking that I messed up the formatting, but that didn't work. I also removed everything from my markdown file except the shortcode, but it didn't help.

Everything else about the property (type, description, etc.) renders correctly. The shortcode is implemented as: `{{< propertylist name=demo sort=name order=asc >}}`. I'm using hugo extended version v0.106.0 and v0.35.8 of the geekdoc theme.  I'm viewing the site locally with the command `hugo serve -D --disableFastRender`.

I was wondering what I was doing wrong?

**Comment by xoxys** [comment]
Hi, please take a look at the breaking changes listed in https://github.com/thegeeklab/hugo-geekdoc/releases/tag/v0.36.0 Using the data file format from the main branch with an older release (< 0.36.0) doesn't work.
